Community.Blazor.MapLibre/Models/Sources/GeoJsonSource.cs

```diff
@@ -18,4 +18,7 @@ public class GeoJsonSource : ISource
     /// </summary>
     [JsonPropertyName("data")]
     public required IFeature Data { get; set; }
+
+    [JsonPropertyName("maxzoom")]
+    public float MaxZoom { get; set; }
```

Comment:
Optional och att den inte kommer med i json alls om den är null, som det är gjort på andra ställen?

```suggestion
    [JsonPropertyName("maxzoom")]
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    public float? MaxZoom { get; set; }
```

Reply:
Viktigt med `[JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]` 👍 . Så ser det inte ut för tex VectorTileSource så den borde inte fungera särskilt bra. null accepteras inte i javascript-koden.